hack/testing/utils

```diff
@@ -76,6 +76,25 @@ try_until_text() {
   log::error "try_until_text expired for '$cmd' == '$expected'"
   return 1
 }
+
+gather_logging_resources() {
+  set +e
```

Comment:
Why bother with `set +e` when each line ends "|| :" which ignores errors anyway?

Reply:
only to ensure that any error in this logic does not fail a test if it otherwise passes.